pkg/yurthub/proxy/util/util.go

```diff
@@ -295,3 +297,22 @@ func WithRequestTimeout(handler http.Handler) http.Handler {
 		handler.ServeHTTP(w, req)
 	})
 }
+
+func WithSaTokenSubsitute(handler http.Handler, token string) http.Handler {
+
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+
+		if info, ok := apirequest.RequestInfoFrom(req.Context()); ok {
+
+			if info.Namespace == "kube-system" && (info.Verb == "get" || info.Verb == "list" || info.Verb == "watch") {
```

Comment:
the condition of replacing bearer token is the bearer token in request header is a bearer token of  kube-system namespace. not the request namespace is kube-system.